### PR TITLE
Add new hooks OnBigWheelWin and CanStudyBlueprint

### PR DESCRIPTION
**Pull request by TurEduard**
I don’t understand what the parameters are and how to fill them in Rust.opj. Can you explain to me?

### CODE REVIEW

**Comment by 2chevskii** [comment]
You'd need to use Oxide.Patcher, located in the corresponding repo. All the instructions are in the same repo in README file

File: resources/Rust.opj [hunk 1]
```diff
@@ -7421,6 +7421,30 @@
...
+              "Exposure": 2,
+              "Name": "Payout",
+              "ReturnType": "System.Void",
+              "Parameters": []
+            },
+            "MSILHash": "epy8ms72cKe1WVahTpdfRH6rY0d+WM136YF5BD/Hcb8=",
+            "BaseHookName": null,
+            "HookCategory": "Entity"
+          }
+        },
```
Comment (TurEduard) [inline resources/Rust.opj]: New hook

**Comment by TurEduard** [comment]
What should I do next?

**Comment by lukespragg** [comment]
Thanks, merging these manually. In the future, please open a separate PR for each hook to avoid any issues if one has changes or isn't one we would accept.

**Pull request closed without merging**